Decred Wallet/DcrlibwalletTypes/Wallet.swift

```diff
@@ -14,19 +14,33 @@ class Wallet: NSObject {
     private(set) var name: String
     private(set) var balance: String
     private(set) var accounts = [DcrlibwalletAccount]()
+    private(set) var visibleAccounts = [DcrlibwalletAccount]()
     private(set) var isSeedBackedUp: Bool = false
     private(set) var displayAccounts: Bool = false
+    private(set) var isRestored: Bool = false
+    private(set) var hasDiscoveredAccounts: Bool = false
+    private(set) var wallet: DcrlibwalletWallet
     
     init(_ wallet: DcrlibwalletWallet) {
         self.id = wallet.id_
         self.name = wallet.name
         self.balance = "\(wallet.totalWalletBalance()) DCR"
         self.accounts = wallet.accounts(confirmations: 0)
+        self.visibleAccounts = self.accounts.filter({!$0.isHidden && $0.number != INT_MAX })
         self.isSeedBackedUp = wallet.seed.isEmpty
         self.displayAccounts = false
+        self.wallet = wallet
     }
     
     func toggleAccountsDisplay() {
         self.displayAccounts = !self.displayAccounts
     }
+    
+    func currentAddress(_ account: Int32, error: NSErrorPointer? = nil) -> String {
+        return self.wallet.currentAddress(account, error: error ?? nil)
+    }
+    
+    func nextAddress(_ account: Int32, error: NSErrorPointer? = nil) -> String {
+        return self.wallet.nextAddress(account, error: error ?? nil)
+    }
```

Comment:
Instead of having alias/helper functions to access the internal wallet methods, use the wallet id where necessary to get a reference to the `DcrlibwalletWallet` object and call the required methods on that object. I'll give an example below.